src/python/ensembl/io/genomio/genome_metadata/prepare.py

```diff
@@ -264,19 +196,13 @@ def main() -> None:
     parser.add_argument_dst_path(
         "--output_file", required=True, help="Output path for the new genome metadata file"
     )
-    parser.add_argument_src_path("--gff3_file", help="GFF3 file to use as annotation source")
-    parser.add_argument(
-        "--base_api_url", default=DEFAULT_API_URL, help="API URL to fetch the taxonomy data from"
+    parser.add_argument_src_path(
+        "--ncbi_meta", required=True, help="JSON file from NCBI datasets for this accession."
```

Comment:
I have removed the reference to `accession` in previous documentation because there is no such argument anywhere.
```suggestion
        "--ncbi_meta", required=True, help="JSON file from NCBI datasets for this genome."
```